src/plugins/ucx/ucx_backend.cpp

```
@@ -133,20 +134,73 @@ class nixlUcxBackendReqH : public nixlBackendReqH {
 
     virtual void
     release() {
-        // TODO: Error log: uncompleted requests found! Cancelling ...
         for (nixlUcxReq req : requests_) {
             const nixl_status_t ret = nixl::ucx::ucsToNixlStatus(ucp_request_check_status(req));
             if (ret == NIXL_IN_PROG) {
-                // TODO: Need process this properly.
-                // it may not be enough to cancel UCX request
+                // A no-op for RMA - ucp_request_cancel() only acts on tag receives - but
```

Cppcheck (2.21.0)

[style] The function 'getMemh' is never used.

(unusedFunction)